changelog/unreleased/issue-3095

```diff
@@ -1,9 +1,17 @@
-Change: Remove `--drive-use-trash=false` from default rclone params
+Change: Deleting files on Google Drive now moves them to the trash
 
-By default restic used launched rclone with --drive-use-trash=false,
-since google drive trash retention policy changed, it is no longer required.
-Rclone will now use what's provided in by the `--drive-use-trash
-parameter, `drive-use-trash` config or RCLONE_CONFIG_*_USE_TRASH env
-falling back to `true` as a default (as of: v1.53.2).
+When deleting files on Google Drive via an rclone backend, restic used
+to bypass the trash folder and required using the `-o rclone.args`
+option to disable this behavior. This ensured that deleted files in
```

Comment:
Everything looks good IMO but I think we should s/disable this behavior/enable the trash folder/ to avoid somewhat of a double negative.

Reply:
Done.